src/udp_wrap.cc

```diff
@@ -516,7 +516,9 @@ void UDPWrap::DoSend(const FunctionCallbackInfo<Value>& args, int family) {
 
   // construct uv_buf_t array
   for (size_t i = 0; i < count; i++) {
-    Local<Value> chunk = chunks->Get(env->context(), i).ToLocalChecked();
+    MaybeLocal<Value> maybe_chunk = chunks->Get(env->context(), i);
+    if (maybe_chunk.IsEmpty()) return;
+    Local<Value> chunk = maybe_chunk.ToLocalChecked();
```

Comment:
```suggestion
    Local<Value> chunk;
    if (chunks->Get(env->context(), i).ToLocal(&chunk))
      return;
```

Reply:
and we should add a test for this case

Reply:
Not really sure about the value added by a test on this really really edge case.